src/components/admin/form-components/SearchableSelector.jsx

```diff
@@ -133,7 +133,7 @@ SearchableSelector.propTypes = {
       id: PropTypes.number.isRequired,
       name: PropTypes.string.isRequired,
     }),
-  ).isRequired,
+  ),
```

Comment:
Based on discussions we have had before on "value" props, this should be required. What's the reason for this change? You mention a bug in SearchableSelector, but don't ever say what that bug is. Is this the fix for that?

Reply:
No I don't think this was the bug, the bug was from a rename of some state where I had missed one some where. 

This and the other weird proptypes related changes (sorry) were proptypes errors I stumbled upon while testing my own code, I think most of them stem from my eslint redo, I think there was an error here because ArticleController passed in `null` when there weren't currently any authors, so I added a default value. But you're right, should I try and change it so that it's required and handling the default in the parent?

Reply:
This one needs reply @Helw150

Reply:
The main point is. Do we allow an empty Selector to be shown as a null or undefined, or do we enforce it should be an empty array? I guess we used empty string in the others, so maybe we should enforce empty array here as well

Reply:
I don't see where in article controller is passing in `null`. On initial load, it passes `[]` and in falcor ToState it passes `_.toArray(article.authors)` which returns `[]` if there are no authors.(actually this might be a fix that only exists on my branch? Not sure). Those are the only places it sets author state other than when `SearchableSelector` calls its `onUpdate`.

That means, if it is passing `null` it's getting null from `SearchableSelector` at some point which we would want to fix. I'm happy if you perform this fix in this branch or if you revert this change and leave it as a *noisy* problem to be fixed. This change just makes it a silent problem though.

Reply:
Ahh no I think I remember. I think that possibly `article.authors` could be null, and then `_.toArray` was throwing errors because it couldn't convert `null` to be an array, that's my best guess anyway. I'm pretty sure there was an error I fixed there anyway. Feel free to test it on your branch

Reply:
No that can't have been it, checked that behaviour

Reply:
Okay maybe I just didn't have a reason lol, I'm gonna test reverting it now

Reply:
Yeah, I tested that behavior before my comment. toArray always returns an array. I think it the problematic functon is `update from 'react-addons-update' in `SearchableSelector`.